justbuild.it-ConsoleApp/src/main/java/com/isa/entity/User.java

```diff
@@ -68,4 +68,9 @@ public String toString() {
                 ", telephoneNumber='" + telephoneNumber + '\'' +
                 '}';
     }
+
+    public String printUser() {
+        return (String.format("{%n          \"firstName\" : \"%s\", %n          \"lastName\" : \"%s\", %n          \"company\" : \"%s\", %n          \"emailAddress\" : \"%s\", %n          \"telephoneNumber\" : \"%s\"}",
+                firstName, lastName, company, emailAddress, telephoneNumber));
+    }
```

Comment:
Ładnie to wygląda teraz w konsoli :)

Reply:
to pewnie można też jakoś wziąć z pliku JSON ale nie wiem jak, i nie bardzo mam czas teraz czytac o tym